src/status_im/contexts/wallet/common/utils/send.cljs

```diff
@@ -0,0 +1,18 @@
+(ns status-im.contexts.wallet.common.utils.send
+  (:require [utils.money :as money]))
+
+(defn calculate-gas-fee
+  [data]
+  (let [gas-amount       (money/bignumber (get data :gas-amount))
+        gas-fees         (get data :gas-fees)
+        eip1559-enabled? (get gas-fees :eip1559-enabled)
+        billion          (money/bignumber "1000000000")]
+    (if eip1559-enabled?
+      (let [base-fee      (money/bignumber (get gas-fees :base-fee))
+            priority-fee  (money/bignumber (get gas-fees :max-priority-fee-per-gas))
+            fee-with-tip  (money/bignumber (money/add base-fee priority-fee))
+            total-gas-fee (money/mul gas-amount fee-with-tip)]
+        (money/with-precision (money/div total-gas-fee billion) 10))
+      (let [gas-price     (money/bignumber (get gas-fees :gas-price))
+            total-gas-fee (money/mul gas-amount gas-price)]
+        (money/with-precision (money/div total-gas-fee billion) 10)))))
```

Comment:
Consider extracting these  "then" and "else" pieces of code to a separate named function, I think it'd do the code easier to pick :+1:

Reply:
Maybe inside those extracted functions we could add a dosctrings that points to the formula to calculate them?
Maybe those should be in the money NS :thinking: 

Another thing is, instead of converting to bignumbers each time, can we convert them once while receiving the data from the back-end?